### PR TITLE
SCALRCORE-21783 Scalr Provider > Token generation

### PR DESCRIPTION
**Pull request by petroprotsakh**
### Changelog
* [x] I have updated CHANGELOG.md according to [CONTRIBUTING.md](../CONTRIBUTING.md)

### Documentation
* [x] I have updated resource/data source documentation in [docs](../docs)

### State
* [ ] My changes affect the state
* [ ] I have included a state migration and a unit test for it

### CODE REVIEW

**Review by artemvang — APPROVED** [review]

File: scalr/resource_scalr_service_account_token.go [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+			if errors.Is(err, scalr.ErrResourceNotFound) {
+				log.Printf("[DEBUG] service account %s not found", saID)
+				d.SetId("")
+				return nil
+			}
+			return diag.Errorf("Error reading service account token %s: %v", id, err)
+		}
+
+		for _, at := range atl.Items {
+			if at.ID == id {
```
Comment (soltysss) [inline scalr/resource_scalr_service_account_token.go:88]: @petroprotsakh if we only need one access token by ID why don't we just use `/service-accounts/<service_account_model:service_account>` endpoint? why it is not published?

**Review by soltysss — APPROVED** [review]

File: scalr/resource_scalr_service_account_token.go (hunk 1 above)
Comment (petroprotsakh) [reply scalr/resource_scalr_service_account_token.go:88]: @soltysss there is no api endpoint for getting single service account's token. `/service-accounts/<service_account_model:service_account>` returns service account, not token.

**Comment by emocharnik** [comment]
@petroprotsakh, Required builds are failing.

**Comment by emocharnik** [comment]
@petroprotsakh, Some non-required builds are failing or stuck.

**Pull request merged**